src/pages/Swap/index.tsx

```diff
@@ -596,6 +605,7 @@ export default function Swap() {
                   allowedSlippage={allowedSlippage}
                 />
               )}
+              {showPriceImpactWarning ? <PriceImpactWarning priceImpact={largerPriceImpact} /> : undefined}
```

Comment:
think it needs to be null rather than undefined? you can also do `showPriceImactWarning && <PriceImpactWarning`

Reply:
apparently in react 18 can return undefined now so can ignore this if you want

Reply:
yeah i tried this and though i was bad at react but actually it just persisted the severity so i needed a new var